lib/ui/text.dart

```diff
@@ -604,7 +609,25 @@ class ParagraphStyle {
   ///   the text.
   ///
   /// * `lineHeight`: The minimum height of the line boxes, as a multiple of the
-  ///   font size.
+  ///   font size. The lines of the paragraph will be at least
+  ///   `(lineHeight + leading) * fontSize` tall when fontSize
+  ///   is not null. When fontSize is null, there is no minimum line height. Tall
+  ///   glyphs due to baseline alignment or large [TextStyle.fontSize] may cause
+  ///   the actual line height after layout to be taller than specified here.
+  ///   [fontSize] must be provided for this property to take effect.
+  ///
+  /// * `leading`: The minimum amount of leading between lines as a multiple of
```

Comment:
Looks like this should be removed too now that leading has been moved to StrutStyle